components/eventing-controller/cmd/upgrade-job/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.16-alpine as builder
+
+ARG DOCK_PKG_DIR=/go/src/github.com/kyma-project/kyma/components/eventing-controller
+
+WORKDIR $DOCK_PKG_DIR
+COPY . $DOCK_PKG_DIR
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o upgradejob ./cmd/upgrade-job
```

Comment:
May be a bit pedantic, but why do we once speak about `upgrade-job` and once about `ubgradejob`?